src/Tribe/Importer/File_Importer_Organizers.php

```diff
@@ -36,22 +36,105 @@ protected function create_post( array $record ) {
 		return $id;
 	}
 
+	/**
+	 * Build a organizer array for creation/update of the current imported organizer.
+	 *
+	 * @since 3.2
+	 * @since TBD Adjust to prevent overwriting values that aren't mapped.
+	 *
+	 * @param int    $organizer_id The ID of the organizer we're currently importing
+	 * @param array  $record       An event record from the import.
+	 *
+	 * @return array $organizer The array of organizer data for creation/update.
+	 */
 	private function build_organizer_array( $organizer_id, array $record ) {
-		$organizer              = array(
-			'Organizer'     => $this->get_value_by_key( $record, 'organizer_name' ),
-			'Description'   => $this->get_value_by_key( $record, 'organizer_description' ),
-			'Email'         => $this->get_value_by_key( $record, 'organizer_email' ),
-			'Phone'         => $this->get_value_by_key( $record, 'organizer_phone' ),
-			'Website'       => $this->get_value_by_key( $record, 'organizer_website' ),
-			'FeaturedImage' => $this->get_featured_image( $organizer_id, $record ),
+		$organizer          = [];
+		$columns        = [
+			'Organizer'   => 'organizer_name',
+			'Description' => 'organizer_description',
+			'Email'       => 'organizer_email',
+			'Phone'       => 'organizer_phone',
+			'Website'     => 'organizer_website',
+		];
+
+		foreach ( $columns as $name => $key ) {
+			// Reset.
+			$value = '';
+
+			// Don't set/overwrite unmapped columns.
+			if ( ! $this->has_value_by_key( $record, $key ) ) {
+				continue;
+			}
+
+			/**
+			 * Allows filtering of main values before setting.
+			 * Return boolean false to prevent importing that value.
+			 * @since TBD
```

Comment:
:no_entry_sign: There must be exactly one blank line before the tags in a doc comment

<sub>hash: cd8d0af8983f5f5fe01b3da188abe789</sub>

Reply:
```suggestion
			 *
			 * @since TBD
```